src/index.ts

```diff
@@ -165,6 +165,9 @@ export function rise(
               body = formatForContentType(body, contenttype);
             }
 
+            if(reqHeaders["Content-Type"] === 'multipart/form-data' && body._boundary)
+              reqHeaders["Content-Type"] = "multipart/form-data; boundary="+body._boundary;
```

Comment:
How do we specify the boundary ? Per the stackoverflow content-type needs to be undefined ?